### DISCUSSION TITLE
Jump to next modified file in tree (Git)

### DISCUSSION THREAD

**Discussion by krmbzds** [nvim-neo-tree/neo-tree.nvim#317] [Q&A] [answered]
I use <kbd>]g</kbd> <kbd>[g</kbd> mappings to jump between hunks. I would like to use similar mappings to jump back and forth between modified git files in neo-tree. I found that nvim-tree has something [like this](https://github.com/kyazdani42/nvim-tree.lua/blob/83fe370d5278db3c967f0d75258487921d23d51e/lua/nvim-tree/actions/init.lua#L39-L40). How can I achieve similar functionality in neo-tree?

![2022-04-22_16-00-51_selection](https://user-images.githubusercontent.com/1383035/164719488-cc62862c-280f-40a0-a80d-c18671feb0b6.svg)

**Comment by nyngwang** [comment]
Personally, I use the default `/`. `<C-D>` and `<C-U>` are also your good friends. But it would still be convenient to have a standalone set of shortcuts for it.

https://user-images.githubusercontent.com/24765272/164756125-3b0942c0-6fea-45be-ad7d-8d3978d519d2.mov

**Comment by cseickel** [comment, marked as answer]
A custom command can be added to do this. The key is that the `state` object that is passed to all commands has a `git_status_lookup` table that you can use to find the "next" file with some git status entry and then focus it. There' enough info scattered around the wiki and these discussions to piece that together.

If you want to open an issue for this, I will create these commands when I have some time.

**Reply by krmbzds** [reply]
Thank you @cseickel for adding this feature!
- https://github.com/nvim-neo-tree/neo-tree.nvim/issues/342#issuecomment-1115550646
